EXILED/Exiled.API/Extensions/RoleExtensions.cs

```diff
@@ -123,6 +123,55 @@ public static bool TryGetRoleBase<T>(this RoleTypeId roleType, out T roleBase)
         /// <returns>Returns whether <paramref name="roleType"/> is an <see cref="IFpcRole"/> or not.</returns>
         public static bool IsFpcRole(this RoleTypeId roleType) => roleType.GetRoleBase() is IFpcRole;
 
+        /// <summary>
+        ///  Checks if the role is an SCP role.
+        /// </summary>
+        /// <param name="roleType">The <see cref="RoleTypeId"/>.</param>
+        /// <returns>A boolean which is true when the role is an SCP role.</returns>
+        public static bool IsScp(this RoleTypeId roleType) => roleType.GetTeam() == Team.SCPs;
+
+        /// <summary>
+        /// Checks if the role is a human role.
+        /// </summary>
+        /// <param name="roleType">The <see cref="RoleTypeId"/>.</param>
+        /// <returns>A boolean which is true when the role is a human role.</returns>
+        public static bool IsHuman(this RoleTypeId roleType) => !roleType.IsDead() && !roleType.IsScp();
+
+        /// <summary>
+        /// Checks if the role is a dead role.
+        /// </summary>
+        /// <param name="roleType">The <see cref="RoleTypeId"/>.</param>
+        /// <returns>A boolean which is true when the role is a dead role.</returns>
+        public static bool IsDead(this RoleTypeId roleType) => roleType.GetTeam() == Team.Dead;
+
+        /// <summary>
+        /// Checks if the role is an NTF role.
+        /// </summary>
+        /// <param name="roleType">The <see cref="RoleTypeId"/>.</param>
+        /// <returns>A boolean which is true when the role is an NTF role.</returns>
+        public static bool IsNtf(this RoleTypeId roleType) => roleType.GetTeam() == Team.FoundationForces;
+
+        /// <summary>
+        /// Checks if the role is a Chaos role.
+        /// </summary>
+        /// <param name="roleType">The <see cref="RoleTypeId"/>.</param>
+        /// <returns>A boolean which is true when the role is an Chaos role.</returns>
+        public static bool IsChaos(this RoleTypeId roleType) => roleType.GetTeam() == Team.ChaosInsurgency;
+
+        /// <summary>
+        /// Checks if the role is a military role (Chaos Insurgency or NTF).
+        /// </summary>
+        /// <param name="roleType">The <see cref="RoleTypeId"/>.</param>
+        /// <returns>A boolean which is true when the role is a military role.</returns>
+        public static bool IsMilitary(this RoleTypeId roleType) => roleType.IsNtf() || roleType.IsChaos();
```

Comment:
currently this includes Guards because of the other thing I mentioned, I think you should specify that this doesn't (or does if it's intended) include them

Reply:
It should grab guards as they are military, but if you want the docs to be changed to mention guards are included in both foundation and military I can do that

Reply:
personally I wouldn't count guards as military, that's why I'd like you to specifically mention if they're counted or not

Reply:
Removing from IsNtf

Reply:
If you could specify in its docs that doesn't include guards that'd be great as well since some people might be confused